PostHog/PostHogRemoteConfig.swift

```diff
@@ -442,6 +453,55 @@ class PostHogRemoteConfig {
         return flags
     }
 
+    func setPersonPropertiesForFlags(_ properties: [String: Any]) {
+        personPropertiesForFlagsLock.withLock {
+            // Merge properties additively, similar to JS SDK behavior
+            for (key, value) in properties {
+                personPropertiesForFlags[key] = value
+            }
+        }
+    }
+
+    func resetPersonPropertiesForFlags() {
+        personPropertiesForFlagsLock.withLock {
+            personPropertiesForFlags.removeAll()
+        }
+    }
+
+    func setGroupPropertiesForFlags(_ groupType: String, properties: [String: Any]) {
+        groupPropertiesForFlagsLock.withLock {
+            // Merge properties additively for this group type
+            if groupPropertiesForFlags[groupType] == nil {
+                groupPropertiesForFlags[groupType] = [:]
+            }
+            for (key, value) in properties {
+                groupPropertiesForFlags[groupType]![key] = value
+            }
+        }
+    }
+
+    func resetGroupPropertiesForFlags(_ groupType: String? = nil) {
```

Comment:
```suggestion
    func resetGroupPropertiesForFlags(_ groupType: String) {
```

you need a type otherwise there's nothing to do, the caller should avoid the call in this case